use-shopping-cart/src/index.js

```diff
@@ -271,7 +271,9 @@ export const useShoppingCart = () => {
       throw new Error('Stripe is not defined');
     }
 
-    const { error } = await stripe.redirectToCheckout(options);
+    const { error } = await stripe.redirectToCheckout(
+      sessionId ? sessionId : options
```

Comment:
Isn't `sessionId` supposed to be an argument to `options`? As far as I can tell from [their docs](https://stripe.com/docs/payments/checkout/one-time) it looks like this line could be:

```suggestion
      sessionId ? { sessionId } : options
```

Reply:
Hitting redirectoToCheckout with a sessionID is separate from hitting it without a sessionId.

The sessionId is a result of passing all the needed options on the server side, so having options with sessionId is actually redundant. Stripe actually throws an error when you try to do this (I found out the hard way lol)

Reply:
Is the documentation on Stripe not updated then because this is their React example?

![image](https://user-images.githubusercontent.com/19195374/80414360-8a00c680-8896-11ea-90ec-7b0c85a71571.png)

Anyway, if it does work with `Stripe#redirectToCheckout(sessionId)` then your code looks good 👍

Reply:
I might not be communicating properly. This example in the Stripe docs is exactly how we're doing it. options are handled on the serverless function. There really is no difference between what we're doing here and what the stripe docs reflect. I'm sorry if I've confused you!


Going to merge!